shared/actions/chat2/index.js

```diff
@@ -1374,7 +1374,7 @@ function* threadSearch(state, action) {
           isRegex: false,
           maxConvsHit: 0,
           maxConvsSearched: 0,
-          maxHits: Constants.inboxSearchMaxTextMessages,
+          maxHits: 1000,
```

Comment:
revert?

Reply:
This is intended, otherwise we only search 25 messages here.